packages/yargs/lib/webpack.js

```diff
@@ -17,32 +55,33 @@ exports.LoggerPlugin = class LoggerPlugin {
       if (this.lastHashes[name] === stats.hash) {
         return;
       }
-      this.lastHashes[name] = stats.hash;
+      const { hash, startTime, endTime } = stats;
+      const isRebuild = this.lastHashes[name];
+      const duration = prettyMS(endTime - startTime);
+
       const hasWarnings = stats.hasWarnings();
       const hasErrors = stats.hasErrors();
-      const duration = prettyMS(stats.endTime - stats.startTime);
+
+      const { assets, errors, warnings, children } = stats.toJson();
+
       if (hasErrors) {
-        this.logger.info(`build target '${name}' failed after ${duration}`);
+        this.logger.info(formatError(name, duration, isRebuild));
       } else {
         if (hasWarnings) {
-          this.logger.info(
-            `build target '${name}' finished with warnings after ${duration}`
-          );
+          this.logger.info(formatWarning(name, duration, assets, isRebuild));
```

Comment:
Same here - **`logger.info` for a warning?

Reply:
Ditto https://github.com/untool/untool/pull/292#discussion_r264287964